src/lib/logical_query_plan/create_view_node.cpp

```diff
@@ -7,28 +7,27 @@
 
 namespace opossum {
 
-CreateViewNode::CreateViewNode(const std::string& view_name, const std::shared_ptr<LQPView>& view)
-    : BaseNonQueryNode(LQPNodeType::CreateView), _view_name(view_name), _view(view) {}
-
-std::string CreateViewNode::view_name() const { return _view_name; }
-
-std::shared_ptr<LQPView> CreateViewNode::view() const { return _view; }
+CreateViewNode::CreateViewNode(const std::string& view_name, const std::shared_ptr<LQPView>& view,
+                               const bool if_not_exists)
+    : BaseNonQueryNode(LQPNodeType::CreateView), view_name(view_name), view(view), if_not_exists(if_not_exists) {}
 
 std::string CreateViewNode::description() const {
   std::stringstream stream;
-  stream << *_view->lqp;
+  stream << *view->lqp;
+  const std::string if_not_exists_output = if_not_exists ? "IfNotExists " : "";
 
-  return "[CreateView] Name: '" + _view_name + "' (\n" + stream.str() + ")";
+  return "[CreateView] " + if_not_exists_output + "Name: '" + view_name + "' (\n" + stream.str() + ")";
```

Comment:
minor: use the stream to build your return value - and use operator<< over operator+ there.